mycroft/skills/settings.py

```diff
@@ -74,8 +76,27 @@
 def get_local_settings(skill_dir, skill_name) -> dict:
     """Build a dictionary using the JSON string stored in settings.json."""
     skill_settings = {}
+    settings_path = None
```

Comment:
Let's not change the logic here, instead determine the correct path in MycroftSkill and use that as an argument